integration-tests/types/config/node/core.go

```diff
@@ -115,13 +115,14 @@ func WithP2Pv1() NodeConfigOpt {
 			ListenIP:   utils2.MustIP("0.0.0.0"),
 			ListenPort: utils2.Ptr[uint16](6690),
 		}
+		// disabled default
```

Comment:
nit, this confused me for a second, maybe change it // P2P is enabled by default.